data/assets/scene/digitaluniverse/pulsars.asset

```diff
@@ -19,40 +19,29 @@ local Object = {
     Type = "RenderablePolygonCloud",
     Enabled = false,
     Labels = {
-      File = speck .. "pulsar.label",
-      Color = { 0.7, 0.2, 0.2 },
-      Size = 15.77,
+      File = speck .. "pulsars.label",
+      Color = { 0.75, 0.21, 0.21 },
+      Size = 15.27,
       MinMaxSize = { 4, 20 },
       Unit = "pc"
     },
     Coloring = {
       FixedColor = { 0.7, 0.0, 0.0 }
     },
     Opacity = 1.0,
-    File = speck .. "pulsar.speck",
+    File = speck .. "pulsars.speck",
     PolygonSides = 4,
     Unit = "pc",
     SizeSettings = {
-      ScaleExponent = 18.4,
+      ScaleExponent = 17.35,
```

Comment:
Same comment here as above.  If we want these to be seen as the same time as the Milky way galaxy, this value should be reset.

Could instead reduce the maxsize if they look too large